### PR TITLE
List User API keys on the web

### PR DESCRIPTION
**Pull request by boddhisattva**
This PR addresses one part of the requirement specified as part of [this issue](https://github.com/hexpm/hexpm/issues/668). Kindly review, Thank you.

Screenshot of how the PR related changes look on the `/dashboard/keys` page:

screen shot 2018-05-05 at 4 55 55 am

### CODE REVIEW

File: lib/hexpm/web/controllers/dashboard_controller.ex [hunk 1]
```diff
@@ -28,6 +28,14 @@ defmodule Hexpm.Web.DashboardController do
     end
   end
 
+  def keys(conn, _params) do
+    user = conn.assigns.current_user
+    keys = Keys.all(user)
+
+    conn
+    |> render_keys(keys)
```
[the remark this answers is not in the document]
Comment (boddhisattva) [reply lib/hexpm/web/controllers/dashboard_controller.ex]: Thanks for your review and kind words @ericmj :). I understand that we could leverage existing Boostrap related CSS and thereby have lesser custom CSS code to add and maintain. I also understand your comment around the usage of single pipes. Thanks for bringing up these things 👍 

I have made changes based on your feedback, please review.

File: test/hexpm/web/controllers/dashboard_controller_test.exs [hunk 2]
```diff
@@ -139,6 +139,15 @@ defmodule Hexpm.Web.DashboardControllerTest do
     assert_delivered_email(Hexpm.Emails.password_changed(c.user))
   end
 
+  test "Show user API keys", c do
+    conn =
+      build_conn()
+      |> test_login(c.user)
+      |> get("dashboard/keys")
+
+      assert response(conn, 200) =~ "API Keys"
```
Comment (wojtekmach) [inline test/hexpm/web/controllers/dashboard_controller_test.exs]: please run `mix format` to fix up indentation

Comment (boddhisattva) [reply test/hexpm/web/controllers/dashboard_controller_test.exs]: Thanks for your feedback @wojtekmach :)  I've updated the code.

**Comment by boddhisattva** [comment]
@er

**Pull request merged**